test/tools/d_do_test.d

```diff
@@ -1597,6 +1731,16 @@ int tryMain(string[] args)
     return 0;
 }
 
+/**
+ * Executes a bash script (deprecated).
```

Comment:
```suggestion
 * Executes a bash script (deprecated). Postprocessors or `dshell` can be used instead.
```

Reply:
Amended to mention `dshell` test as `POST_SCRIPT` is up for deprecation as well